crates/store/re_types/definitions/rerun/archetypes/line_strips2d.fbs

```diff
@@ -4,10 +4,10 @@ namespace rerun.archetypes;
 
 /// 2D line strips with positions and optional colors, radii, labels, etc.
 ///
-/// \example archetypes/line_strip2d_simple !api image="https://static.rerun.io/line_strip2d_simple/c4e6ce937544e66b497450fd64ac3ac2f244f0e1/1200w.png"
+/// \example archetypes/line_strips2d_simple !api image="https://static.rerun.io/line_strip2d_simple/c4e6ce937544e66b497450fd64ac3ac2f244f0e1/1200w.png"
 /// \example archetypes/line_segments2d_simple !api image="https://static.rerun.io/line_segment2d_simple/53df596662dd9ffaaea5d09d091ef95220346c83/1200w.png"
```

Comment:
hmmm the arcehtype is still `line_strip2d`, so maybe this one whould be `line_strip2d_segments_simple`?